modules/azure_functions/src/azure_functions.c

```diff
@@ -304,7 +304,15 @@ static void AzureFunctions_Receive(MODULE_HANDLE moduleHandle, MESSAGE_HANDLE me
         else
         {
             /* Codes_SRS_AZUREFUNCTIONS_04_013: [ AzureFunctions_Receive shall base64 encode by calling Base64_Encode_Bytes, if it fails it shall fail and return. ] */
-            STRING_HANDLE contentAsJSON = Base64_Encode_Bytes(content->buffer, content->size);
+			STRING_HANDLE contentAsJSON;
+			if (content->buffer == NULL)
+			{
+				contentAsJSON = STRING_construct_n("", 0);
+			}
+			else
+			{
+				contentAsJSON = Base64_Encode_Bytes(content->buffer, content->size);
+			}
             if (contentAsJSON == NULL)
```

Comment:
Suggest to put a line between this so it's easier to read (easy to mix with elseif if it's together.